src/main/java/com/sprint/mission/discodeit/service/basic/BasicChannelService.java

```diff
@@ -0,0 +1,157 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.dto.*;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ChannelType;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.MessageRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.repository.ReadStatusRepository;
+import com.sprint.mission.discodeit.service.ChannelService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.*;
+
+@Service
+@RequiredArgsConstructor
+public class BasicChannelService implements ChannelService {
+    private final ChannelRepository channelRepository;
+    private final MessageRepository messageRepository;
+    private final ReadStatusRepository readStatusRepository;
+    private final UserRepository userRepository;
+
+    @Override
+    public ChannelDto create(ChannelDto dto, List<UUID> userList) {
+        //private 신청 검토
+        Channel savedChannel = channelRepository.save(dto.type());
+        userList.forEach(userId -> readStatusRepository.save(new ReadStatusDto(null, userId, savedChannel.getId(), null, null, null)));
+        return new ChannelDto(savedChannel);
+    }
+
+    @Override
+    public ChannelDto create(ChannelDto dto) {
+        //public 신청 검토
+        Channel savedChannel = channelRepository.save(dto.name(), dto.type());
+        //readStatus 업데이트 필요
+
+        return new ChannelDto(savedChannel);
+    }
+
+    @Override
+    public ChannelFindDto find(UUID id) {
+        Channel channel = channelRepository.findById(id);
+        if(channel == null){
+            throw new IllegalStateException("채널을 찾을 수 없습니다.");
+        }
+        List<Message> messagesById = messageRepository.findMessagesById(id);
```

Comment:
messageRepository.findMessagesById 살펴보니까 사용자 id를 파라미터로 받고, 사용자가 작성한 모든 메시지 가져오는걸로 구현이 되어 있네요. 채널에 존재하는 메시지를 가져와야 할거 같아요.

추가로, 아래와 같이 엔티티 이름을 명시해주면 로직 읽기가 더 쉬워질듯 합니다. 어떤 id인지 구현체를 봐야 알게 되네요ㅜㅜ
```suggestion
        List<Message> messagesById = messageRepository.findMessagesByChannelId(id);
```